src/frontend/src/btc/types/network.ts

```diff
@@ -0,0 +1,3 @@
+import type { Network, NetworkAppMetadata } from '$lib/types/network';
+
+export type BitcoinNetwork = Network & Partial<Pick<NetworkAppMetadata, 'explorerUrl'>>;
```

Comment:
For now, I think it is safe to remove the Pick part, right? Just an observation, unless you plan already on adding more things there that will be non-optional